packages/generator/src/artifacts/types/token-types.ts

```diff
@@ -52,9 +52,9 @@ export function generateTokenTypes(ctx: Context) {
       result.add(`\t\t${key}: ${typeName}Token`)
     }
 
-    const keyframes = Object.keys(theme?.keyframes ?? {})
+    const keyframes = unionType(Object.keys(theme?.keyframes ?? {}))
 
-    set.add(`export type AnimationName = ${unionType(keyframes)}`)
+    if (keyframes) set.add(`export type AnimationName = ${keyframes}`)
```

Comment:
```suggestion
    if (theme?.keyframes) set.add(`export type AnimationName = ${unionType(Object.keys(theme.keyframes))}`)
```

You can remove L55 with this